modules/swagger-core/src/main/java/io/swagger/util/PropertyDeserializer.java

```diff
@@ -250,6 +248,11 @@ Property propertyFromNode(JsonNode node) {
                 arrayProperty.setMinItems(getInteger(node, PropertyBuilder.PropertyId.MIN_ITEMS));
                 arrayProperty.setMaxItems(getInteger(node, PropertyBuilder.PropertyId.MAX_ITEMS));
                 arrayProperty.setUniqueItems(getBoolean(node, PropertyBuilder.PropertyId.UNIQUE_ITEMS));
+                JsonNode example = getDetailNode( node, PropertyBuilder.PropertyId.EXAMPLE);
+                if( example != null ) {
+                    arrayProperty.setExample( Json.mapper().convertValue(example, Object.class));
```

Comment:
not this is the right type of value to set as the example; we could also simply save the JsonNode!?